jujugui/static/gui/src/app/state/test-state.js

```diff
@@ -1359,9 +1359,16 @@ describe('State', () => {
         location: {href: '/u/hatch/staging/i/applications/inspector/ghost'}
       });
       const pushStub = sinon.stub(state, '_pushState');
-      state.dispatch();
+      state.bootstrap();
```

Comment:
These should have been bootstrap all along. If it was, this issue would have been caught much sooner.